tests/acceptance/21_methods/action_ifelapsed_respected.cf.sub

```diff
@@ -0,0 +1,23 @@
+bundle agent main
+{
+  methods:
+    "test"
+      action => expireafter_ten_sec;
+}
+
+body action expireafter_ten_sec
+{
+  expireafter => "10";
```

Comment:
BTW the filename and the pull request title mention "ifelapsed" test, but this is about expireafter.